freebridge-ai/test_rag_flow.py

```
@@ -0,0 +1,263 @@
+        sql_project = "INSERT INTO projects (employer_id, freelancer_id, job_posting_id, project_name, status, headcount, created_at, updated_at) VALUES (%s, %s, %s, '금융권 프로젝트', 'COMPLETED', 1, %s, %s)"
+        self.cursor.execute(sql_project, (self.employer_id, self.freelancer_id, self.job_id, now, now))
+
+    def tearDown(self):
+        # 각 테스트 종료 후 실행
+        # 1. DB 롤백 (INSERT 했던 데이터 모두 취소되고, DELETE 했던 기존 데이터 복구됨)
+        self.conn.rollback()
+        
+        # 2. ChromaDB 테스트 컬렉션 삭제 (데이터 정리)
+        try:
+            self.vectorstore.delete_collection()
+        except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)